web/packages/extension/build/_locales/sv/messages.json

```diff
@@ -0,0 +1,56 @@
+{
+    "settings_ruffle_enable": {
+        "message": "Spela upp Flash-innehåll i Ruffle"
+    },
+    "settings_page_ignore_optout": {
+        "message": "Spela upp Flash-innehåll även på sidor som inaktiverar Ruffle"
+    },
+    "settings_ignore_optout": {
+        "message": "Ignorera webbsidekompatibilitetsvarningar"
+    },
+    "status_init": {
+        "message": "Avläser aktuella fliken…"
+    },
+    "status_no_tabs": {
+        "message": "Det finns ingen aktiv flik."
+    },
+    "status_tabs_error": {
+        "message": "Ett fel uppstod när den aktuella fliken slogs upp."
+    },
+    "status_message_init": {
+        "message": "Kontrollerar Ruffle-status i aktuella fliken…"
+    },
+    "status_result_running": {
+        "message": "Ruffle är loadad och spelar upp Flash-innehåll i den aktuella filken."
+    },
+    "status_result_optout": {
+        "message": "Ruffle är inte laddad för den nuvarande sidan har markerat sig själv som inkompatibel."
+    },
+    "status_result_disabled": {
+        "message": "Ruffle är inte laddad för den stängdes av av användaren."
+    },
+    "status_result_error": {
+        "message": "Ett fel uppstod letade upp den aktuella flikens instans av Ruffle."
+    },
+    "status_result_protected": {
+        "message": "Ruffle kan inte ladda på skyddade webbläsarsidor."
+    },
+    "action_reload": {
+        "message": "Ladda om fliken för att applicera ändringar"
+    },
+    "open_settings_page": {
+        "message": "Öppna inställningarna"
+    },
+    "settings_page": {
+        "message": "Inställningar"
+    },
+    "description": {
+        "message": "Puts Flash back on the web where it belongs."
```

Comment:
Completely missing translation.
"Sätter tillbaka Flash på webbsidan/hemsidan den tillhör." I'm frankly not sure what word is most popular in 2020, you choose.